controllers/openstackcluster_controller.go

```diff
@@ -618,6 +672,12 @@ func reconcileNetworkComponents(scope *scope.WithLogger, cluster *clusterv1.Clus
 		return fmt.Errorf("failed to reconcile network: ManagedSubnets only supports one element, %d provided", len(openStackCluster.Spec.ManagedSubnets))
 	}
 
+	err = reconcileLoadBalancerNetwork(openStackCluster, networkingService)
```

Comment:
This isn't exactly "reconcile", more of a "resolve". Not 100% sure, so @mdbooth might have a better opinion.

Reply:
Yeah, this is a resolve rather than a reconcile. I'm not going to get hung up on that, though, especially as I'm not 100% convinced of 'resolve' myself.